x-pack/plugins/cloud_security_posture/public/pages/configurations/configurations.tsx

```diff
@@ -16,17 +16,15 @@ import { useDataView } from '../../common/api/use_data_view';
 import { cloudPosturePages, findingsNavigation } from '../../common/navigation/constants';
 import { LatestFindingsContainer } from './latest_findings/latest_findings_container';
 import { DataViewContext } from '../../common/contexts/data_view_context';
-import { useLatestFindings } from './latest_findings/use_latest_findings';
 
 export const Configurations = () => {
   const location = useLocation();
   const dataViewQuery = useDataView(CDR_MISCONFIGURATIONS_DATA_VIEW_ID_PREFIX);
   const { data: getSetupStatus, isLoading: getSetupStatusIsLoading } = useCspSetupStatusApi();
-  const getLatestFindings = useLatestFindings({ sort: [['asc']], enabled: true, pageSize: 1 });
-  const hasFindings = !!getLatestFindings.data?.pages?.[0]?.total;
+  const hasMisconfigurationsFindings = !!getSetupStatus?.hasMisconfigurationsFindings;
 
-  const hasConfigurationFindings =
-    hasFindings ||
+  const hasFindings =
+    hasMisconfigurationsFindings ||
     getSetupStatus?.kspm.status === 'indexed' ||
```

Comment:
nit: I think `getSetupStatus?.kspm.status === 'indexed' || getSetupStatus?.cspm.status === 'indexed'` check is not needed anymore, I can't think of a scenario when there are no documents in the CDR index pattern, but either kspm or cspm is `indexed`